### PR TITLE
[Doc] Fix the doc for the partition_graph function

### PR DESCRIPTION
**Pull request by TristonC**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented
- [x] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Comment by TristonC** [comment]
@zheng-da

**Comment by jermainewang** [comment]
@zheng-da  and @classicsong may not be able to review. Reassigned to @Rhett-Ying .

**Comment by Rhett-Ying** [comment]
@dgl-bot

File: python/dgl/distributed/partition.py [hunk 1]
```diff
@@ -532,7 +532,7 @@ def partition_graph(g, graph_name, num_parts, out_path, num_hops=1, part_method=
     ...                                 out_path='output/', reshuffle=True,
     ...                                 balance_ntypes=g.ndata['train_mask'],
     ...                                 balance_edges=True)
-    >>> g, node_feats, edge_feats, gpb, graph_name = dgl.distributed.load_partition(
+    >>> g, node_feats, edge_feats, gpb, graph_name, ntypes_list, etypes_list = dgl.distributed.load_partition(
```
Comment (Rhett-Ying) [inline python/dgl/distributed/partition.py:535]: this line is too long, could you keep each line not longer than `80` characters?

**Review by Rhett-Ying — APPROVED** [review]
could you pls keep each line not longer than `80` characters? Others look good to me.

**Comment by TristonC** [comment]
@Rhett-Ying Is the CI broken or there is a lint issue?

**Comment by Rhett-Ying** [comment]
@dgl-bot

**Pull request merged**